test/testautomation_stdlib.c

```diff
@@ -1228,6 +1228,123 @@ stdlib_strpbrk(void *arg)
     }
     return TEST_COMPLETED;
 }
+
+#undef SDL_wcstol
```

Comment:
Do you need this undef?

Reply:
I don't actually know, but some of the other tests in this file have `#undef SDL_strnlen` above them so I took the safe route and copied them.

Reply:
I think you can remove these entirely. This was from an older version of SDL that defined public symbols in terms of their C runtime counterparts.